src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -759,6 +751,14 @@ layout network::get_node_output_layout(const primitive_id& output_id) const {
     return (*res)->get_node_output_layout();
 }
 
+memory::ptr network::get_output_memory(const primitive_id& id) {
+    return get_primitive(id)->output_memory_ptr();
+}
+
+layout network::get_output_layout(const primitive_id& id) const {
+    return get_primitive(id)->get_output_layout();
+}
+
```

Comment:
why did you change this?

Reply:
`output_id` is not proper parameter name for this function